pb/api/v1/data.proto

```diff
@@ -254,3 +258,46 @@ message FailureStats {
   int32 num_failing_tests = 1;
 }
 
+// Summarized representation of data from tab's HealthinessInfo.
+// Will be rendered in healthiness summary component within tab summary.
+message HealthinessSummary {
+
+  // Top flaky tests (with +ve flakiness) by the current flakiness %.
```

Comment:
nit: Sorry, can you clarify what "with +ve flakiness" means?

Reply:
Sure, tests with flakiness > 0. I will also update the description